chasm/test_component_test.go

```diff
@@ -20,12 +20,13 @@ type (
 	TestComponent    struct {
 		UnimplementedComponent
 
-		ComponentData     *protoMessageType
-		SubComponent1     Field[*TestSubComponent1]
-		SubComponent2     Field[*TestSubComponent2]
-		SubData1          Field[*protoMessageType]
-		SubComponents     Map[string, *TestSubComponent1]
-		PendingActivities Map[int, *TestSubComponent1]
+		ComponentData         *protoMessageType
+		SubComponent1         Field[*TestSubComponent1]
+		SubComponent2         Field[*TestSubComponent2]
+		SubData1              Field[*protoMessageType]
+		SubComponents         Map[string, *TestSubComponent1]
+		PendingActivities     Map[int, *TestSubComponent1]
+		SubComponent11Pointer Field[*TestSubComponent11]
```

Comment:
I am actually debating with myself if we need another `chasm.Ptr` type. I feel the current approach is more flexible and allow Component author to define only one field and have logic like, if there's already a Component in the tree (say from its parent), just points to it, otherwise create a real one. 

If there's need a for making things more explicit we can add `chasm.Ptr` later and it's essentially a Field with type pointer.